source/postprocess/visualization.cc

```diff
@@ -63,13 +63,29 @@ namespace aspect
             const unsigned int n_q_points = uh.size();
             for (unsigned int q=0; q<n_q_points; ++q)
               for (unsigned int i=0; i<computed_quantities[q].size(); ++i)
-                computed_quantities[q][i]=uh[q][i] * ((i < dim) ? velocity_scaling_factor : 1.0);
+                {
+                  if (this->introspection().component_masks.velocities[i] ||
```

Comment:
Uh, could you add a comment that explains this condition? My head is spinning while trying to get the meaning. Probably you want the velocity in the one-phase case and the fluid velocity in the two-phase case, right?